src/upf_network.py

```diff
@@ -21,10 +21,12 @@ class UPFNetworkError(Exception):
 class NetworkInterface:
     """A class to interact with a network interface."""
 
-    def __init__(self, name: str):
+    def __init__(self, name: str, ip_address: str, mtu_size: int = 1500):
```

Comment:
Similar comment, we probably don't want to have default values hardcoded in multiple places